Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/inbound/StandardInboundDocQueryTest.java

```diff
@@ -95,12 +101,24 @@ public void failedPolicy() {
 
         when(policyChecker.checkIncomingPolicy(request, assertion)).thenReturn(false);
 
+        assertion.setDeferredResponseEndpoint("http://deferredEnpoint/");
```

Comment:
Why are we modifying this test? Deferred logic should only run when the endpoint is provided - logic should remain unchanged if the deferred endpoint is not a part of the assertion.

You should have a new JUnit test to test your new functionality.